index.html

```diff
@@ -2,95 +2,167 @@
+					img/home/home-villas-img.png 1024w"
+			sizes = "(max-width: 500px) 500px,
+					1024px"
+			src="img/home/home-villas-img.png" alt="Second slide">
+		<h3 class="section-title one">THE VILLAS</h3>
+		<div class="text-content">
+			<p>The Villas bring to the table win-win survival strategies to ensure proactive domination. At the end of the day, going forward,
+			a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution.</p><br>
+			<p>Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. Override the digital divide with
+			additional clickthroughs from DevOps. Nanotechnology immersion along the information highway will close the loop on focusing
+			solely on the bottom line.</p>
+		</div>
+	</section>
+	<section class="content-stacked">
+		<img
+			srcset= "img/home/home-mobile-outskirts-img.png 500w,
+					  img/home/home-outskirts-img.png 1024w"
+			sizes="(max-width: 500px) 500px,
+					1024px"
+		src="img/home/home-outskirts-img.png" alt="Second slide">
+		<h3 class="section-title two">OUTSKIRTS</h3>
+		<div class="text-content">
+			<p>The Outskirts are amazing to the table win-win survival strategies to ensure proactive domination. At the end of the day,
+			going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution.</p><br>
+			<p>Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. Override the digital divide with
+				additional clickthroughs from DevOps. Nanotechnology immersion along the information highway will close the loop on focusing
+				solely on the bottom line.</p>
+		</div>
+	</section>
+	<section class="content-stacked">
+		<img
+			srcset="img/home/home-mobile-the-blocks-img.png 500w,
+					img/home/home-the-blocks-img.png 1024w"
+			sizes="(max-width: 500px) 500px,
+					1024px" 
+			src="img/home/home-the-blocks-img.png" alt="Second slide">
+		<h3 class="section-title three">THE BLOCKS</h3>
+		<div class="text-content">
+			<p>The Blocks are amazing to the table win-win survival strategies to ensure proactive domination. At the end of the day, going
+			forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution.</p><br>
+			<p>Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. Override the digital divide with
+				additional clickthroughs from DevOps. Nanotechnology immersion along the information highway will close the loop on focusing
+				solely on the bottom line.</p>
+		</div>
+	</section>
+</div>
+<footer class="footer">
+	<form class="form">
+		<h5>Interested in starting a project?
+		<br>Let’s talk:</h5>
+		<input type="text" name="  email" value="Enter email">
+		<p>We'll never share your email with anyone else.</p>
+	</form>
+	<address class="address">
+		<h5>New York</h5>
```

Comment:
You could just target this with the address tag instead of giving it a class name.